packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/DrawingToolMarks/DrawingToolMarks.js

```diff
@@ -42,12 +42,19 @@ function DrawingToolMarks ({ activeMarkId, marks, onDelete, onDeselectMark, onFi
     }
 
     function deselectMark (event) {
-      onDeselectMark(mark)
```

Comment:
[cleanup] It seems that the `onDeselectMark` event hook is being removed here, which is fine, because it doesn't seem to be used. (If some future tool needs it, we can always add it back.)

If that's the case, won't it make sense to remove the `DrawingToolMarks.onDeselectMark` property altogether?

It's declared in the constructor, propTypes, defaultProps, and the test specs but not actually invoked anywhere; and heck, I can't seem to find the term `onDeselectMark` anywhere else in the `lib-classifier`.

Reply:
I think you're absolutely right `onDeselectMark` isn't used, but I think for now I'd like to add the hook back in for consistency between DrawingToolMarks and InteractionLayer, though structure `deselectMark` similar to `endMoveMark`. Maybe we can make a point to review/refactor/cleanup the drawing pieces once at a somewhat complete state?

Reply:
👍